pytissueoptics/scene/intersection/__init__.py

```diff
@@ -1,2 +1,3 @@
 from .ray import Ray
+from .raySource import RaySource, UniformRaySource, RandomPositionAndOrientationRaySource
```

Comment:
We will soon move Ray and sources to its own module

Reply:
Soon